Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -8,6 +8,7 @@ import SnapKit
 private let ToolbarBaseAnimationDuration: CGFloat = 0.2
 
 class TabScrollingController: NSObject {
+    private var profile: Profile
```

Comment:
Since the profile won't be changed, we don't need to make this a mutable `var` and can use a `let` instead.